query/expression.go

```diff
@@ -46,34 +116,34 @@ type EvaluationContext struct {
 // Expressions correspond to the timerange in the current EvaluationContext.
 type Expression interface {
 	// Evaluate the given expression.
-	Evaluate(context EvaluationContext) (*api.SeriesList, error)
+	Evaluate(context EvaluationContext) (Value, error)
+}
+
+func EvaluateToSeriesList(e Expression, context EvaluationContext) (api.SeriesList, error) {
```

Comment:
I added a new commit where I unexported `value`, `seriesListValue`, `stringValue`, `scalarValue`, and `conversionError`.